tests/ccl_test_distances.py

```diff
@@ -368,7 +363,7 @@ def compare_distances_mnu(z, chi_bench,dm_bench, Omega_v, w0, wa, Neff, mnu):
 
     assert_allclose(dm, dm_bench[a_not_one], atol=1e-3, rtol = DISTANCES_TOLERANCE_MNU)
 
-@decorators.slow
+@numpy.testing.dec.slow
```

Comment:
Have you checked slow/fast tests still run after this change?

Reply:
yes it appears to work.